plugins/Live/Reports/GetLastVisitsDetailsWidget.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Copyright (C) Piwik PRO - All rights reserved.
+ *
+ * Using this code requires that you first get a license from Piwik PRO.
+ * Unauthorized copying of this file, via any medium is strictly prohibited.
+ *
+ * @link http://piwik.pro
+ */
+
+namespace Piwik\Plugins\Live\Reports;
+
+use Piwik\Plugins\Live\Visualizations\VisitorLogWidget;
+
+class GetLastVisitsDetailsWidget extends Base
```

Comment:
This would possibly add it to ReportMetadata and break APIs / Apps (eg Piwik Mobile app) which cannot handle such a custom format of a report. I haven't tested it but I can imagine it will break stuff. 

It might help to define the method

```php
    public function configureReportMetadata(&$availableReports, $infos)
    {
    }
```
I'd recommend to test it with mobile app . Also we need to check re other side effects.